docs/src/starknet/index.md

```diff
@@ -51,14 +50,16 @@ response: [0x0]
 > In the above example we supply `sncast` with `--account` and `--url` flags. If `snfoundry.toml` is present, and have these properties set, values provided using these flags will override values from `snfoundry.toml`. Learn more about `snfoundry.toml` configuration [here](../projects/configuration.md#sncast).
 
 
-### Calldata
+### Arguments and Calldata
```

Comment:
How about we change this section to `### Arguments` and explain only how to use `--arguments` flag.

Below, we can add a section `### Using Serialized Calldata` and keep the docs for `--calldata`.